tutorials/text/annotations.py

```diff
@@ -47,22 +47,45 @@
 # 'data'              use the axes data coordinate system
 # ==================  ========================================================
 #
-# For example to place the text coordinates in fractional axes
-# coordinates, one could do::
+# The following strings are also valid arguments for *textcoords*
 #
-#     ax.annotate('local max', xy=(3, 1),  xycoords='data',
-#                 xytext=(0.8, 0.95), textcoords='axes fraction',
-#                 arrowprops=dict(facecolor='black', shrink=0.05),
-#                 horizontalalignment='right', verticalalignment='top',
-#                 )
+# ==================  ========================================================
+# argument            coordinate system
+# ==================  ========================================================
+# 'offset points'     offset (in points) from the xy value
+# 'offset pixels'     offset (in pixels) from the xy value
+# ==================  ========================================================
 #
 # For physical coordinate systems (points or pixels) the origin is the
-# bottom-left of the figure or axes.
+# bottom-left of the figure or axes. Points are
+# `typographic points <https://en.wikipedia.org/wiki/Point_(typography)>`_
+# meaning that they are a physical unit measuring 1/72 of an inch. Points and
+# pixels are discussed in further detail in :ref:`transforms-fig-scale-dpi`.
 #
-# Optionally, you can enable drawing of an arrow from the text to the annotated
-# point by giving a dictionary of arrow properties in the optional keyword
-# argument *arrowprops*.
+# .. _annotation_data:
+#
+# Annotating data
+# ~~~~~~~~~~~~~~~
+#
+# For example to place the text coordinates in fractional axes
+# coordinates, one could do ::
+
+fig, ax = plt.subplots()
+ax.scatter(3, 1, s=20)
+ax.annotate('local max', xy=(3, 1),  xycoords='data',
+             xytext=(0.8, 0.95), textcoords='axes fraction',
+             horizontalalignment='right', verticalalignment='top')
```

Comment:
I don't think you want this `::` here with real code following. This treats the next line generated by sphinx-gallery as a code block, instead of the code:
![image](https://user-images.githubusercontent.com/302469/185022105-c557c52d-9658-4cc3-9054-1ed6a3f77fe4.png)